README.md

```diff
@@ -93,6 +93,7 @@ Following inputs can be used as `step.with` keys:
 
 | Name             | Type    | Default                            | Description                                   |
 |------------------|---------|------------------------------------|-----------------------------------------------|
+| `scan-type`      | String  | `image`                           | Scan type, e.g. `image`,`fs` or `repo` 
```

Comment:
```suggestion
| `scan-type`      | String  | `image`                           | Scan type, e.g. `image`,`fs` or `repo` (default: `image`)
```

Reply:
I think we should remove `repo` from this. It doesn't make much sense to run that as part of a Github Action.